pkg/specgen/specgen.go

```diff
@@ -523,8 +523,8 @@ type ContainerHealthCheckConfig struct {
 // swagger:model SpecGenerator
 type SpecGenerator struct {
 	ContainerBasicConfig
-	ContainerStorageConfig
-	ContainerSecurityConfig
+	ContainerStorageConfig  `json:"container_storage_config,omitempty"`
+	ContainerSecurityConfig `json:"container_security_config,omitempty"`
```

Comment:
Why the change here? I think this will break backwards compat. Spegcen must be stable.